master/internal/configpolicy/utils.go

```diff
@@ -44,6 +44,39 @@ func ValidWorkloadType(val string) bool {
 	}
 }
 
+func UnmarshalConfigPolicies[T any](errMsg string, constraints,
+	invariantConfig *string) (*model.Constraints, *T,
+	error,
+) {
+	var globalConstraints *model.Constraints
+	var globalConfig *T
```

Comment:
Maybe call the function arguments "constraintStr" and "configStr" and these just "constraints" and "config"? I'm not fussed about the actual names, but I think it's confusing to use "global" here.

Reply:
Definitely makes sense to change these var names, done!